app/modules/loan/endpoints_loan.py

```diff
@@ -767,6 +796,28 @@ async def update_loan(
             quantity=quantity,
         )
         await cruds_loan.create_loan_content(loan_content=loan_content, db=db)
+        end_loan_template = templates.get_template("activation_mail.html").render(
+            {
+                "loan_detail": ", ".join(
+                    [str(item.total_quantity) + " " + item.name for item in loan.items],
+                ),
+                "loaner": loan.loaner.name,
+            },
+        )
+        await scheduler.cancel_job(job_id=f"loan_end_{loan.id}")
```

Comment:
I think you could put this in the `if settings.SMTP_ACTIVE` block, as there is no real gain in getting the template if we don't send the mail